index.html

```diff
@@ -440,94 +442,96 @@
                However in Polymer 1.2 there is no dynamic type injection for
                DOM stamping... oh well.
               -->
-          <template is="dom-if" class="tpl learn-more" if="[[showPage('learn-more', tab)]]">
+          <template is="dom-if" id="learn-more-template" if="[[showPage('learn-more', tab)]]">
```

Comment:
@juliocri I like the pattern and approach you developed.

I prefer to use `id` for identify an element instead of assigning a class; when I read through the code, an assigned class always makes me think there is special CSS somewhere for that class (for example `tpl`)

So I changed your code to use an `id` instead of `class`, and updated the routines using it. Are you OK with that?